appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: '{build}'
+
+image: Visual Studio 2017
+
+# Test against the latest version of this Node.js version
+# environment:
+#   nodejs_version: "6"
+
+platform:
+  - x64
+
+cache:
+  - node_modules
+  - '%APPDATA%\npm-cache'
+
+init:
+  - git config --global core.autocrlf input
```

Comment:
The AppVeyor reference (https://www.appveyor.com/docs/appveyor-yml/) lists this as an example of “scripts that are called at very beginning, before repo cloning”.  I’ve seen it used in several examples in my build testing, e.g., slqectron-gui (https://github.com/sqlectron/sqlectron-gui/blob/master/appveyor.yml), Demo_electron_update (https://github.com/vijayantkatyal/Demo_electron_update/blob/master/appveyor.yml), BlankUp-Electron (https://github.com/HoverBaum/BlankUp-Electron/blob/master/appveyor.yml), vicatcu/Electron-Sandbox (https://github.com/vicatcu/Electron-Sandbox/blob/master/appveyor.yml), and nodecg/dashboard (https://github.com/nodecg/dashboard/blob/master/appveyor.yml).  Electron also uses this config (https://github.com/electron/electron/blob/master/appveyor.yml).